### ISSUE TITLE
Large image in README

### ISSUE THREAD

**Issue by kwangure** [finos/perspective#1422]
It's unkind to force someone just visiting your project to immediately download [a 9MB GIF](https://github.com/finos/perspective/blob/gh-pages/img/demo_large.gif). This holds for both your README and [website](https://perspective.finos.org/#web-browser).

I'd remove the GIF entirely from the README given that you have a wonderful collection of examples below it. If you choose to keep it, consider shortening the duration of the GIF and nesting it in a `<details></details>` toggle to make it opt-in. Otherwise, it blocks the initial page load.

Consider using a video instead (i.e `<video autoplay loop muted playsinline></video>`) on the website. See [web.dev](https://web.dev/replace-gifs-with-videos/) for more info/ideas on optimizing GIFs.

**Comment by timkpaine** [comment]
Did you test your proposals on github readmes? They don't support generic html, they are sanitized.

**Comment by texodus** [comment]
@kwangure I appreciate the feedback, but I strongly favor keeping this as I believe it does an exceptional job introducing what Perspective is capable of in the most compact delivery possible.  Regarding the technical aspect:

* [GitHub Allowed tag list](https://github.com/github/markup/issues/245#issuecomment-682231577) does not include `<video>`.
* `<img>` tags don't block page load (though I realize large images reduce overall ttl).
* GitHub markdown already limits assets to `10mb`.
* AFAIK, only GIF is allowed on gfh-markdown.  This was the case when the `README.md` was written at least, though there seems to be some indication other pages e.g. issues support `.mp4` and `.webm` [now](https://github.blog/changelog/2020-12-16-video-upload-public-beta/).
* I assure you, this GIF has already been optimized to the extent of my ability and patience.

I'll leave this open for now - if this is a common sentiment, we can set aside a some dev time in the future to revisit this.

**Comment by kwangure** [comment]
I'm aware that Github does not allow the `<video>` tag. Note that I suggested using it on your website (i.e [perspective.finos.org](https://perspective.finos.org/)) and not your README. For Github, I suggested making the GIF initially hidden and "togglable" using the `<details>` tag. 

In my original comment, I linked to the [#web-browser](https://perspective.finos.org/#web-browser) section of your landing page, which uses a ~5MB GIF that could be `<video>`fied. My previous comment is not as clear as it could have been.
